src/raftstore/store/config.rs

```diff
@@ -138,6 +141,7 @@ pub struct Config {
 impl Default for Config {
     fn default() -> Config {
         Config {
+            sync_log: false,
```

Comment:
I don't think it's a good idea to set the default value to false. And it's conflict with the [description](https://github.com/pingcap/tikv/pull/1840/files#diff-d4c79fcc9faffa48ae170c90d8c0683aR52).